<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>959</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>WaveBuzzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/lorbeer_64x64.png</normaloff>:/icons/lorbeer_64x64.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back/Frame_1920x1080_sw_dark.jpg);
background-repeat: repeat-xy;
background-origin: content;
background-position: center center;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>220</number>
    </property>
    <property name="topMargin">
     <number>160</number>
    </property>
    <property name="rightMargin">
     <number>220</number>
    </property>
    <property name="bottomMargin">
     <number>240</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageTimer">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>60</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>60</number>
        </property>
        <property name="bottomMargin">
         <number>60</number>
        </property>
        <item>
         <widget class="QLabel" name="labelCurrentTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>SF Square Root Shaded</family>
            <pointsize>200</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar {
background: #333;
border-radius: 20px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #2c2,
stop: 0.5 #191,
stop: 1 #060 );
border-radius: 20px;
}</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageScores">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetTeamA" native="true">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="labelLorbeerTeamA">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>128</height>
              </size>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTeamA">
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>44</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border: 0px;
	background: transparent;
}</string>
             </property>
             <property name="text">
              <string>Team 1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTeamAScore">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>180</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: transparent;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetTeamB" native="true">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="labelLorbeerTeamB">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>128</height>
              </size>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTeamB">
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>44</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border: 0px;
	background: transparent;
}</string>
             </property>
             <property name="text">
              <string>Team 2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTeamBScore">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>180</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: transparent;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetTeamC" native="true">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelLorbeerTeamC">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>128</height>
              </size>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTeamC">
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>44</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border: 0px;
	background: transparent;
}</string>
             </property>
             <property name="text">
              <string>Team 3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTeamCScore">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>180</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: transparent;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetTeamD" native="true">
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="labelLorbeerTeamD">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>160</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>128</height>
              </size>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTeamD">
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>44</pointsize>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border: 0px;
	background: transparent;
}</string>
             </property>
             <property name="text">
              <string>Team 4</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTeamDScore">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>DejaVu Sans</family>
               <pointsize>180</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background: transparent;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
